pkg/auth/manager.go

```diff
@@ -52,7 +53,7 @@ type authResult struct {
 	ci             *monitor.ConnectionInfo
 	localIdentity  identity.NumericIdentity
 	remoteIdentity identity.NumericIdentity
-	remoteHostIP   net.IP
+	remoteNodeID   uint16
```

Comment:
This is only created & used locally to the agent right? (Just checking since I believe these are not synchronized across the cluster)